ui/packages/catalog-ui-search/src/main/webapp/component/select/select.collection.view.js

```diff
@@ -161,7 +161,10 @@ define([
                     }).length > 1))) {
                 return false;
             }
-            if (this.getAppropriateString(child.get('value')).indexOf(filterValue) === -1) {
+            if (child.get('label') !== undefined && this.getAppropriateString(child.get('label')).indexOf(filterValue) === -1) {
```

Comment:
Can we do this instead of 2 if statements?
```
    if (child.get('label') === undefined || this.getAppropriateString(child.get('value')).indexOf(filterValue) === -1) {
        return false;
    }
```

Reply:
I don't think that's equivalent. We're doing child.get('label') above and not 'value'